test/Itinero.Test/LocalGeo/CoordinateTests.cs

```diff
@@ -84,5 +84,18 @@ public void TestGeoCoordinateOffsetEstimate()
             Assert.AreEqual(distance, distanceLat, 0.3);
             Assert.AreEqual(distance, distanceLon, 0.3);
         }
+
+        [Test]
+        public void TestGeoCoordinateDistanceEstimateInMeter()
+        {
+            var coordinateStart = new Coordinate(179f, 0f);
+
+            var coordinateEnd = new Coordinate(-179f, 0f);
```

Comment:
These are `LocalGeo` coordinates, so the parameter order is actually `latitude` then `longitude`.  -179 / 179 are not valid latitude values.  Did you mean to reverse the order?

Reply:
Yes sorry I work in Normalized terms usually (X,Y) or (Lon, Lat)

Reply:
Yes I usually work in (X,Y) or (Lon,Lat) The code should be correct but but the parameters to `Coordinate` should be reverse as your indicated:

```
 [Test]
        public void TestGeoCoordinateDistanceEstimateInMeter()
        {
            var coordinateStart = new Coordinate(0f, 179f);

            var coordinateEnd = new Coordinate(0f, -179f);

            var distance = Coordinate.DistanceEstimateInMeter(coordinateStart, coordinateEnd);

            Assert.IsTrue(distance == 222581.84375);
        }
```